content/en/blog/_posts/0000-00-00-kube-proxy-non-privileged.md

```diff
@@ -44,8 +44,10 @@ made, but with `--init-only` you can have `kube-proxy` itself do the setup
 
 ## Initializing kube-proxy in an init container
 
-Usually, cluster operators run `kube-proxy` in a privileged security context. Here's a snippet of an
-example manifest:
+*The example manifests below are not complete, but narrowed down to what is
+essential to illustrate the function.*
```

Comment:
Thanks, but I think we can safely assume that a good proportion of readers' eyes slip right past this and start at the manifest.

Reply:
(we should keep the note here, but we should also make it clear in the example manifest about which details aren't included)